src/components/app/Root.jsx

```diff
@@ -35,23 +32,8 @@ class Root extends React.Component {
                 />
 
                 <Route
-                    path="/sync/organisationUnits"
-                    render={props => <OrganisationUnitPage d2={d2} {...props} />}
-                />
-
-                <Route
-                    path="/sync/dataElements"
-                    render={props => <DataElementPage d2={d2} {...props} />}
-                />
-
-                <Route
-                    path="/sync/indicators"
-                    render={props => <IndicatorPage d2={d2} {...props} />}
-                />
-
-                <Route
-                    path="/sync/validationRules"
-                    render={props => <ValidationRulePage d2={d2} {...props} />}
```

Comment:
In the future we might consider: ``/sync/metadata/indicators`` or other routes programmatically and add a way to preload the filter. But not in this PR for sure, just leaving the comment for future.